src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -149,6 +149,17 @@ public ChassisSpeeds getTargetSpeeds(
         Constants.Drive.MAX_SPEED);
   }
 
+  /**
+   * Sets the drive mode in smart dashboard.
+   *
+   * <p><strong> THIS DOES NOT SET THE ROBOT'S DRIVE MODE</strong>
+   *
+   * @param driveMode the mode to display in smart dashboard.
+   */
+  public static void setDriveMode(String driveMode) {
```

Comment:
Don't make this static; by making it only accessible from the singleton instance you'll make it a bit less likely that it'll get called from the wrong place. Also making static methods in subsystems is weird.